backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -73,7 +74,16 @@ object VeloxBackendSettings extends BackendSettingsApi {
       format: ReadFileFormat,
       fields: Array[StructField],
       partTable: Boolean,
+      rootPaths: Seq[String],
       paths: Seq[String]): ValidationResult = {
```

Comment:
What's the differences between `rootPaths` and `paths`?

Reply:
There is a discussion about this: https://github.com/apache/incubator-gluten/pull/4086#discussion_r1432313011

Reply:
paths is emty for velox, as intializing it is time consuming op.
https://github.com/apache/spark/blob/b54b3006f94604c41f230cb94fa1ef86da8ff6c9/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/PartitioningAwareFileIndex.scala#L114C16-L114C26